PROTOCOL.md

```diff
@@ -420,6 +421,7 @@ The following is an example `add` action:
     "dataChange": true,
     "baseRowId": 4071,
     "defaultRowCommitVersion": 41,
+    "clusteringProvider": "liquid",
```

Comment:
this is an invalid scenario where both `partitionValues` and `clusteringProvider` are set?

Reply:
Good catch! I added a json example for clusetred table only.

Reply:
Ah I think we should still have the `"partitionValues":{}`.

Reply:
Confirmed: [AddFile.partitionValues](https://github.com/delta-io/delta/blob/master/spark/src/main/scala/org/apache/spark/sql/delta/actions/actions.scala#L674-L675) is always in the json
```scala
    @JsonInclude(JsonInclude.Include.ALWAYS)
    partitionValues: Map[String, String],
```

Reply:
Thanks and I fixed the clustered table json content with an empty partitionValues